rules/Transform/Rector/Attribute/AttributeKeyToClassConstFetchRector.php

```diff
@@ -72,20 +80,64 @@ class SomeClass
      */
     public function getNodeTypes(): array
     {
-        return [Attribute::class];
+        return [
+            Class_::class,
+            Property::class,
+            Param::class,
+            ClassMethod::class,
+            Function_::class,
+            Closure::class,
+            ArrowFunction::class,
+            Interface_::class,
+        ];
     }
 
     /**
-     * @param Attribute $node
+     * @param Class_|Property|Param|ClassMethod|Function_|Closure|ArrowFunction|Interface_ $node
```

Comment:
I think we should have an `AttributeAwareInterface`, to allow catching these with single type.
Could you add it to patches the way we have `StmtsAwareInterface`?

Reply:
I will try, for note, some of the list is already `StmtsAwareInterface` patched, so patch sorting may be important to apply correct patch